src/test/java/edu/ucsb/mapache/google/ItemTests.java

```diff
@@ -0,0 +1,57 @@
+package edu.ucsb.mapache.google;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+
+public class ItemTests {
+    @Test
+    public void test_to_string() throws Exception {
+        Item item = new Item("test", "test", "test", "test", "test", "test", "test", "test", "test", "test");
+        String expected = "{" +
+        " kind='" + "test" + "'" +
+        ", title='" + "test" + "'" +
+        ", htmlTitle='" + "test" + "'" +
+        ", link='" + "test" + "'" +
+        ", displayLink='" + "test" + "'" +
+        ", snippet='" + "test" + "'" +
+        ", htmlSnippet='" + "test" + "'" +
+        ", cacheId='" + "test" + "'" +
+        ", formattedUrl='" + "test" + "'" +
+        ", htmlFormattedUrl='" + "test" + "'" +
+        "}";
+        assertEquals(expected, item.toString());
+    }
+
+    @Test
+    public void test_equals0() throws Exception {
+        Item item0 = new Item("test", "test", "test", "test", "test", "test", "test", "test", "test", "test");
+        Item item1 = new Item("test", "test", "test", "test", "test", "test", "test", "test", "test", "test");
+        assertTrue(item0.equals(item1));
+        assertTrue(item0.equals(item0));
+    }
+
+    @Test
+    public void test_equals1() throws Exception {
+        Item item0 = new Item("test", "test", "test", "test", "test", "test", "test", "test", "test", "test");
+        String[] arg = {"test", "test", "test", "test", "test", "test", "test", "test", "test", "test"};
+        for(int i = 0; i<10; i++){
```

Comment:
Minor nitpick, but instead of `10` if you did `arg.length()` it'd make it clearer that you're iterating through the arg array.

Reply:
Good point. The idea here is that 10 corresponds to the number of arguments in the constructor, not the arg array itself since the argument might not be of the same type and multiple arg arrays may be required. This is the case for QueriesTests.